apps/foundation-demo/src/app/checkout/components/shipping/shipping-options/shipping-options.component.spec.ts

```diff
@@ -55,12 +55,36 @@ describe('ShippingOptionsComponent', () => {
 
   describe('when radio is clicked', () => {
     
-    it('should call onSelectShippingOption', () => {
+    it('should not call onSelectShippingOption', () => {
```

Comment:
Why?

Reply:
there's a bug where if you click between the radio input (the circle thing) and the label, the radio will not actually be selected. It's just how the html radio input works I guess.

Reply:
Ok. I was reading the spec directly and not looking at the component. Can we make the spec more descriptive? The intent is not obvious from what's written.

Reply:
done